queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/*Find all animals whose name ends in "mon"*/ 
+SELECT * From animals WHERE name LIKE '%mon';
+/*List the name of all animals born between 2016 and 2019*/
+SELECT name From animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+/*List the name of all animals that are neutered and have less than 3 escape attempts*/
+SELECT name from animals where neutered=true and escape_attempts>3;
```

Comment:
- [ ] Dear @Victor-chiemerie , here we are interested in animals having less than 3 escape attempts, hence we should query for escape_attempts < '3'.  Please keep in mind its very important to follow along the given requirements.